### DISCUSSION TITLE
Is it possible to translate slugs?

### DISCUSSION THREAD

**Discussion by drpiou** [expo/router#852] [Q&A]
Hello 👋

In the process of choosing expo-router in my expo app (web and mobile), I am wondering if it is possible to have translated slugs to match the file system:

```
app/
  account/
    index.tsx
  posts/
    index.tsx
```

- /en/account -> `app/account/index.tsx`
- /fr/compte -> `app/account/index.tsx`

**Comment by marklawlor** [comment]
Expo Router doesn't currently support built-in internationalisation but we would like to explore it more in the future

**Reply by drpiou** [reply]
Thank you! In the meantime I'll use an another approach then. I'll keep an eye on next updates hoping this will be implemented. Best :)
